default/stringtables/en.txt

```diff
@@ -5047,6 +5056,69 @@ SITREP_EMPIRE_ELIMINATED_LABEL
 Empire Eliminated
 
 #############################################################
+####           RANDOM BEGINNER HINTS                     ####
+#############################################################
+BEGINNER_HINTS
+Random Game Hints
+
+RANDOM_BEGINNER_HINT
+%rawtext%
+
+BEGINNER_HINT_01
+A good first step is to build scout ships and explore the region around your home system.
+
+BEGINNER_HINT_02
+If you can conquer planets with other species and add them to your empire, it may be easier for them to colonize different environments.  Not all native species can colonize and some can not build ships.
+
+BEGINNER_HINT_03
+Ships will be repaired in a single turn at a system that has a friendly [[buildingtype BLD_SHIPYARD_ORBITAL_DRYDOCK]] if there are no enemies in the system that turn.
+
+BEGINNER_HINT_04
+Systems that are connected by thick lines are [[encyclopedia SUPPLY_TITLE]] line linked and they share [[encyclopedia INDUSTRY_TITLE]] Production Points (PP).
+
+BEGINNER_HINT_05
+All system share [[encyclopedia RESEARCH_TITLE]] Points (RP) that they generate even if they are not connected by supply lines. If you have a remote system that is unconnected, setting the [[encyclopedia FOCUS_TITLE]] to research might be a more efficient use.
+
+BEGINNER_HINT_06
+Ships will refuel when they are in a system that is connected by thick or by thin lines, as long as the thin lines extend back to thick supply line connected systems.
```

Comment:
rephrase "thin lines"... a bit unclear what it means, and only applicable to an empire's own supply connected systems.